### ISSUE TITLE
Crash while riding past some acidic zombies

### ISSUE THREAD

**Issue by l29ah** [CleverRaven/Cataclysm-DDA#38556]
# Describe the bug

```
#0  0x00007f31f7cd271b in raise () from /lib64/libc.so.6
#1  0x00007f31f7cba55d in abort () from /lib64/libc.so.6
#2  0x000055fadc2e7899 in signal_handler (sig=11) at src/crash.cpp:121
#3  <signal handler called>
#4  std::accumulate<int const*, int> (__init=0, __last=0x90, __first=__first@entry=0x68) at /usr/lib/gcc/x86_64-pc-linux-gnu/8.3.0/include/g++-v8/bits/stl_numeric.h:120
#5  dealt_damage_instance::total_damage (this=this@entry=0x68) at src/damage.cpp:179
#6  0x000055fadc9cdefa in mdefense::acidsplash (m=..., source=0x55fadf91dc60, proj=0x0) at src/mondefense.cpp:92
#7  0x000055fadc9f582a in monster::on_hit (this=0x55faf12bcfc0, source=0x55fadf91dc60, proj=0x0) at src/monster.cpp:2683
#8  0x000055fadc2ef0c9 in Creature::deal_melee_hit (this=0x55faf12bcfc0, source=0x55fadf91dc60, hit_spread=<optimized out>, critical_hit=<optimized out>, dam=..., dealt_dam=...) at src/creature.cpp:517
#9  0x000055fadca0cd2f in monster::melee_attack (this=0x55fadf91dc60, target=..., accuracy=<optimized out>) at src/monster.cpp:1268
#10 0x000055fadc9e55a5 in monster::attack_at (p=..., this=0x55fadf91dc60) at src/monmove.cpp:1363
#11 monster::attack_at (this=0x55fadf91dc60, p=...) at src/monmove.cpp:1331
#12 0x000055fadc9f0b46 in monster::move (this=this@entry=0x55fadf91dc60) at src/monmove.cpp:776
#13 0x000055fadc47505b in game::monmove (this=0x55fadf6cad50) at src/game.cpp:4128
#14 0x000055fadc4769da in game::do_turn (this=0x55fadf6cad50) at src/game.cpp:1524
#15 0x000055fadbc8e221 in main (argc=<optimized out>, argv=<optimized out>) at /usr/lib/gcc/x86_64-pc-linux-gnu/8.3.0/include/g++-v8/bits/unique_ptr.h:342
```

# Steps To Reproduce

dunno

# Versions and configuration

todays master

# Additional context

**Comment by anothersimulacrum** [comment]
Fixed by #38555 ?

**Comment by l29ah** [comment]
I assume yes and will report if it occurs again, thanks.
